README.md

```diff
@@ -35,12 +35,16 @@ Edit this document to include your answers after each question. Make sure to lea
 1. What is the difference between an adaptive website and a fully responsive website?
 
```

Comment:
Ah it looks like you forgot this one.  Can you leave a comment here with your answer?